<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoughParamWindow</class>
 <widget class="QMainWindow" name="HoughParamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hough Parameter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Hough</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>171</width>
         <height>527</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="enableHoughCirclesCheckBox">
          <property name="text">
           <string>enable Hough Circles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="maxRadiusBox">
          <property name="title">
           <string>maxRadius</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QSpinBox" name="maxRadiusSpinBox">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="minRadiusBox">
          <property name="title">
           <string>minRadius</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QSpinBox" name="minRadiusSpinBox">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="param2Box">
          <property name="title">
           <string>param2</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="param2DoubleSpinBox">
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="dpBox">
          <property name="title">
           <string>dp</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="dpDoubleSpinBox">
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="param1Box">
          <property name="title">
           <string>param1</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="param1DoubleSpinBox">
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="minDistBox">
          <property name="title">
           <string>minDist</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="minDistDoubleSpinBox">
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>pixel size (mm)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="pixelSizeDoubleSpinBox">
             <property name="maximum">
              <double>99990000.000000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>enableHoughCirclesCheckBox</zorder>
      <zorder>verticalLayoutWidget</zorder>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Blur</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>20</y>
         <width>171</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="enableBlurCheckBox">
          <property name="text">
           <string>enable Blur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="blurKSizeBox">
          <property name="title">
           <string>BlurKSize</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QSpinBox" name="blurKSizeSpinBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="blurSigmaBox">
          <property name="title">
           <string>BlurSigma</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QSpinBox" name="blurSigmaSpinBox"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
